ecosystem_cicd_tools/release.py

```diff
@@ -51,7 +51,7 @@ def get_commit(commit_id=None, repo=None):
     return repo.get_commit(commit_id)
 
 
-def create_release(name, version, message, commit, repo=None):
+def create_release(name, version, message, commit="master", repo=None):
```

Comment:
This isn't necessary, because we grab the commit ID from environment variables...for local testing, we probably need to find another method. (like grabbing from `git log`)

Reply:
this commit used for "target_commitish" and in our case it will be always be "master" because we  create new version from the master code.

Reply:
No it will not always be master.